script.js

```diff
@@ -1,121 +1,138 @@
-  function drawNumbers(ctx, radius) {
-    var ang;
-    var num;
-    ctx.font = radius * 0.15 + "px arial";
-    ctx.textBaseline = "middle";
-    ctx.textAlign = "center";
-    for(num = 1; num < 13; num++){
-      ang = num * Math.PI / 6;
-      ctx.rotate(ang);
-      ctx.translate(0, -radius * 0.85);
-      ctx.rotate(-ang);
-      ctx.fillText(num.toString(), 0, 0);
-      ctx.rotate(ang);
-      ctx.translate(0, radius * 0.85);
-      ctx.rotate(-ang);
-    }
-  }
-//   Get's the time and then draws it on the canvas
-  function drawTime(ctx, radius){
-    var now = new Date();
-    var hour = now.getHours();
-    var minute = now.getMinutes();
-    var second = now.getSeconds();
-    //hour
-    hour = hour%12;
-    hour = (hour*Math.PI/6)+(minute*Math.PI/(6*60))+(second*Math.PI/(360*60));
-    drawHand(ctx, hour, radius*0.5, radius*0.07);
-    //minute
-    minute = (minute*Math.PI/30)+(second*Math.PI/(30*60));
-    drawHand(ctx, minute, radius*0.8, radius*0.07);
-    // second
-    second = (second*Math.PI/30);
-    drawHand(ctx, second, radius*0.9, radius*0.02);
-  }
+  hour = hour%12;
+  hour = hourPosition(hour, minute,second);
+  minute = minutePosition(minute, second);
+  second = secondPosition(second);
 
-  function drawHand(ctx, pos, length, width) {
-    ctx.beginPath();
-    ctx.lineWidth = width;
-    ctx.lineCap = "round";
-    ctx.moveTo(0,0);
-    ctx.rotate(pos);
-    ctx.lineTo(0, -length);
-    ctx.stroke();
-    ctx.rotate(-pos);
-  }
+  drawHand(ctx, hour, radius*0.5, radius*0.07);
+  drawHand(ctx, minute, radius*0.8, radius*0.07);
+  drawHand(ctx, second, radius*0.9, radius*0.02);
+}
 
+//Hour hand position
+function hourPosition(hours, minutes, seconds){
+  return (hours*Math.PI/6)+(minutes*Math.PI/(6*60))+(seconds*Math.PI/(360*60))
+}
+//Minute hand position
+function minutePosition(minutes, seconds){
+  return (minutes*Math.PI/30)+(seconds*Math.PI/(30*60))
+}
+//Second hand position
+function secondPosition(seconds) {
+  return (seconds*Math.PI/30)
+}
 
-// Digital Clock
+//Draws clocks hands
+function drawHand(ctx, pos, length, width) {
+  ctx.beginPath();
+  ctx.lineWidth = width;
+  ctx.lineCap = "round";
+  ctx.moveTo(0,0);
+  ctx.rotate(pos);
+  ctx.lineTo(0, -length);
+  ctx.stroke();
+  ctx.rotate(-pos);
+}
 
-  function currentTime() {
-    var date = new Date(); /* creating object of Date class */
-    var hour = date.getHours();
-    var min = date.getMinutes();
-    var sec = date.getSeconds();
-    hour = updateTime(hour);
-    min = updateTime(min);
-    sec = updateTime(sec);
-    document.getElementById("clock").innerText = hour + " : " + min + " : " + sec; /* adding time to the div */
-      var t = setTimeout(function(){ currentTime() }, 1000); /* setting timer */
+//Grabs the current time for digital clock
+function digitalClock() {
+  var date=new Date(); 
+  var hour=date.getHours();
+  var min=date.getMinutes();
+  var sec=date.getSeconds();
+  hour=updateTime(hour);
+  min=updateTime(min);

```

Comment:
i'm confused here, why does update time take in an int but then is the same for hour/min/sec? seems wrong